lib/spack/spack/compiler.py

```diff
@@ -174,6 +176,105 @@ def check(exe):
     def version(self):
         return self.spec.version
 
+    # This property should be overridden in the compiler subclass if
+    # a verbose flag is available
+    @property
+    def verbose_flag(self):
+        # If it is not overridden, assume it is not supported
+        pass
+
+    _LINKER_LINE = \
+        re.compile(r'^( *|.*[/\\])'
+                   r'(link|ld|([^/\\]+-)?ld|collect2)'
+                   r'[^/\\]*( |$)')
```

Comment:
This looks dumb because `flake8` is picky.

Reply:
I usually avoid the `\` by breaking after the paren, then indent the contents by 4.

Reply:
Yeah, that still requires a line break though since the `re.compile(…)` hits 80 with the extra indentation.